homeassistant/components/tuya/translations/de.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
These files don't matter. They will be automatically deleted when the translation updater runs because our translations are stored in lokalise. That's where they will need to be added.

Reply:
Sorry, I don’t particularly understand what this means. These are the translations in our scripts. After deleting, where do the scripts get the translations?

Reply:
Our infrastructure will fetch the translations from Lokalise and include them in the source.

There's a script that can be run to include the default English translation to ease testing during development. The other translations json files should be removed.

```py
python -m script.translations develop
```

Reply:
I’m sorry I’m not very familiar with Lokalise. How do I add translations and associate them with my script? I just tested the following script. I can get our English language pack, but we haven’t uploaded it yet. Why
`python -m script.translations develop`
`Integration to process: tuya`

Reply:
You are not expected to do translation in Lokalise for this PR. If you want to help with translation you can do that but it's not required for this PR to be merged. The community will help with translation in Lokalise. How to help with translation is described in the docs I linked.

The only required file for this PR is the `strings.json` file that is the template for the translations.

Reply:
OK，thanks